pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -74,6 +74,17 @@ func (v *ControlPlaneMutator) Handle(ctx context.Context, req admission.Request)
 		}
 	}
 
+	// As we are deprecating IOR, on creating a v2.5 SMCP we want to disable IOR if not specified explicitly
+	if req.AdmissionRequest.Operation == admissionv1beta1.Create {
+		newOpenShiftRoute := mutator.NewOpenShiftRoute()
+
+		if newOpenShiftRoute == nil {
+			if versions.V2_5.Version().String() == currentVersion {
```

Comment:
If a new SMCP is created without providing a version, `currentVersion` here will be an empty string. Because of this, IOR will not be disabled when the version is not specified and the default version is 2.5.

Also, should versions greater than 2.5 be considered here too?